packages/pwa/app/commerce-api/hooks/useShopper.js

```diff
@@ -15,11 +15,10 @@ import {customerProductListTypes} from '../../constants'
  * when the app loads on the client-side. Should only be use at top-level of app.
  * @returns {Object} - customer and basket objects
  */
-const useShopper = () => {
+const useShopper = (currency) => {
```

Comment:
Similar to how `currency` was optional in `getOrCreateBasket` we should probably do that here. It will work without it right?

Thinking about it a little more, maybe `getOrCreateBasket` shouldn't have the options object as descripted above. But pass the options to the hook like this:


```
const basket = useBasket({currency: 'USD'})
```

And the `getOrCreateBasket` will have access to it from it's scope.

Reply:
Doing it this way will reduce how often you have to pass around the `currency` like in the 2 calls below.